osfs/os.go

```diff
@@ -168,6 +168,42 @@ func (fs *OS) RemoveAll(path string) error {
 	return os.RemoveAll(fullpath)
 }
 
+// Symlink creates newname as a symbolic link to oldname.
+// All parent directories are created.
+func (fs *OS) Symlink(oldname, newname string) error {
```

Comment:
IMHO `link` and `target` are very good names, they are domain-accurate and are commonly used in manpages and other documentation.